tests/helix/send-to-helix-buildonhelixtests.targets

```diff
@@ -30,15 +30,16 @@
     <ItemGroup>
       <_TestRunCommandArguments Remove="@(_TestRunCommandArguments)" />
 
+      <!-- TODO: Fix this for MTP -->
       <_TestBlameArguments Remove="@(_TestBlameArguments)" />
       <_TestBlameArguments Include="--blame-crash" />
       <_TestBlameArguments Include="--blame-crash-dump-type full" />
 
       <!-- Using `dotnet test` for the project directly here -->
-      <_TestRunCommandArguments Include="dotnet test -s .runsettings --results-directory $(_HelixLogsPath) -v:n" />
+      <_TestRunCommandArguments Include="dotnet exec @.runsettings --results-directory $(_HelixLogsPath)" />
```

Comment:
this is finding .runsettings next to the test csproj?

Reply:
I think next to test executable. And that will be there because you already have this in tests/Directory.Build.targets:

```
    <None Include="$(RunSettingsFilePath)" CopyToOutputDirectory="PreserveNewest" Condition="'$(DeployRunSettingsFile)' == 'true'" />

```